db/external_sst_file_ingestion_job.cc

```diff
@@ -1067,7 +1068,7 @@ IOStatus ExternalSstFileIngestionJob::GenerateChecksumForIngestedFile(
       &file_checksum, &file_checksum_func_name,
       ingestion_options_.verify_checksums_readahead_size,
       db_options_.allow_mmap_reads, io_tracer_, db_options_.rate_limiter.get(),
-      ro);
+      ro, nullptr /* stats */, nullptr /* clock */);
```

Comment:
Same

Reply:
I don't think we have external sst ingestion read stats yet but will do this fix in advance!

Reply:
Fixed